static/js/login_or_signup.js

```diff
@@ -149,7 +149,7 @@ $(function() {
     });
 
     attachSubmitClickHandler(loginFormMetadata, (formDataAsJsonString) => {
-        $.post('/session', formDataAsJsonString, (responseBody, _, request) => {
+        $.post('/api/session', formDataAsJsonString, (responseBody, _, request) => {
```

Comment:
it sucks that there is no way to export all the urls in that we have in our paths, so that they can be accessed through javascript.

Reply:
i'm sure there is a way to do it, though it may have to be kinda hacky. We ran into this issue at bevspot and we did some silly shit to get it working the way we wanted

Reply:
definitely something we can do, just not a priority